package.json

```diff
@@ -37,5 +37,8 @@
   },
   "resolutions": {
     "@nuxt/fonts": "0.11.4"
+  },
+  "dependencies": {
+    "nuxt-charts": "^0.2.3"
```

Comment:
Frontend deps should be in frontend/package.json. Also please remove old chart.js packages